sorting/sorting.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Println("The program sorts three numbers.")
+	fmt.Println("Enter the numbers:")
+	var a, b, c float64
+	fmt.Scanln(&a, &b, &c)
+	if a > b && a > c {
```

Comment:
nit: I recommend all of these conditions look the same. You keep changing `<` and `>` the same would be:
```go
if a > b && a > c { // a is the largest
  ...
} else if b > a && b > c { // b is the largest 
  ...
} else { // we could assume c is the largest
  ...
}
```